coordinator-external/src/main/scala/com/socrata/datacoordinator/external/DataCoordinatorError.scala

```diff
@@ -0,0 +1,110 @@
+package com.socrata.datacoordinator.external
+
+sealed class DataCoordinatorError
```

Comment:
Ohh I see, I should have taken a closer look.  Thank you for describing the design choice.  Changing this would be more of a design pivot other then an optimization or refactoring.  So I vote it is not worth it if external clients are satisfied.